src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestMethodCommand.class.st

```diff
@@ -0,0 +1,98 @@
+"
+Description
+--------------------
+
+I am a command available in the method context menu of Calypso. When activated I'll jump on the test method corresponding to the method. 
+
+If the method does not exists I will create it in the corresponding class.
+
+Internal Representation and Key Implementation Points.
+--------------------
+
+    Instance Variables
+	selectedMethods:			<aCollection>	Methods selected by the user that should have their tests generated.
+	testMethodToBrowse:		<aMethod>			The method to browse at the end of the execution of the command.
+
+"
+Class {
+	#name : #ClyJumpToTestMethodCommand,
+	#superclass : #CmdCommand,
```

Comment:
There is SycMethodCommand for multiple method operations. 
But maybe we should simplify this logic for single method? (And introduce SycSingleMethodCommand which does not exist yet)

Reply:
There is no problem here since we can select multiple methods to generate multiple tests.